packages/contracts-bedrock/contracts/test/FaultDisputeGame.t.sol

```diff
@@ -27,7 +24,7 @@ contract FaultDisputeGame_Test is DisputeGameFactory_Init {
     /**
      * @dev The current version of the `FaultDisputeGame` contract.
      */
-    string internal constant VERSION = "0.0.1";
+    string internal constant VERSION = "0.0.2";
```

Comment:
Don't think this is used anywhere?

Reply:
See L89

Reply:
Ah missed because of gh collapse, thanks!